app/gui/qt/load_source_dialog.cpp

```diff
@@ -4,7 +4,6 @@
 #include <QIODevice>
 #include <fstream>
 #include <iostream>
-#include <QDebug>
```

Comment:
This is from your suggestion at the time @tombettany. Debts paid :)

Reply:
Touché